.travis.yml

```diff
@@ -38,15 +39,13 @@ env:
   - EMBER_TRY_SCENARIO=default
   - EMBER_TRY_SCENARIO=ember-release
   - EMBER_TRY_SCENARIO=ember-beta
```

Comment:
I'd vote to remove beta as well, it ties up our limited CI builds more and I've already removed it from a few repos such as `ember-frost-bunsen`.